Src/BootCamp.Chapter/LogUtility/FileLogger.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.IO;
+
+namespace BootCamp.Chapter.LogUtility
+{
+    class FileLogger : ILogger
+    {
+        readonly string _logPath;
+        public FileLogger(string LogPath)
+        {
+            _logPath = LogPath;
+        }
+
+        public void Log(string msg)
+        {
+            File.AppendAllText(_logPath, msg + "\r\n");
```

Comment:
Instead of \r\n use System.Environment.NewLine. This is better because it will make your code independent of the environment. \r\n is valid only on windows I think.

Reply:
Replaced "\r\n" with System.Environment.NewLine.  Thank you for letting me know about this NewLine.